charts/argo-cd/values.yaml

```diff
@@ -37,6 +37,10 @@ crds:
 
 ## Globally shared configuration
 global:
+  # -- Default domain used by all components
+  ## Used for ingresses, certificates, SSO, notifications, etc.
+  domain: argocd.example.com
```

Comment:
Can be also hostname if you want. We have a mix between ingresses / certificates / urls.